components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/constant/ActionMgtConstants.java

```diff
@@ -25,6 +25,7 @@ public class ActionMgtConstants {
 
     public static final String URI_ATTRIBUTE = "uri";
     public static final String AUTHN_TYPE_ATTRIBUTE = "authnType";
+    public static final String IDN_SECRET_TYPE_ACTION_SECRETS = "ACTION_SECRET_PROPERTIES";
 
```

Comment:
What about just saying 'ACTION_SECRETS'.
What's the pattern followed commonly

Reply:
Followed the pattern used in 
https://github.com/wso2/carbon-identity-framework/blob/5bb95d04dbe7e9a9864af78358ef62a399729648/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql#L1126C1-L1127C156

Reply:
ACTION_API_ENDPOINT_AUTH_SECRETS